src/schema/basic_slots.yaml

```diff
@@ -264,14 +264,41 @@ slots:
     domain: PlannedProcess
     range: Protocol
 
-
   #  nucleic_acid_concentration:
   #    range: InstrumentValue
 
   biomaterial_purity:
     domain: ProcessedSample
     range: QuantityValue
 
+  qc_failure_what:
+    domain: FailureCategorization
+    range: FailureWhatEnum
+    description: >-
+      Provides a summary about what caused a lab or workflow process to fail
+    comments:
+      - For example Low read count from a sequencer, malformed fastq files, etc)
+
+  qc_failure_where:
+    domain: FailureCategorization
+    range: FailureWhereEnum
+    description: >-
+      Describes the nmdc schema class that corresonds to where the failure occurred.
+      Most commonly this would be the same as Class that generated the results.
+    comments:
+      - If the assembly size was too small to proceed to annotation failure_where would be MetagenomeAssembly.
+
+  qc_comment:
+    range: string
+    description: >-
+      Slot to store additional comments about laboratory or workflow output. For workflow output
+      it may describe the particular workflow stage that failed. (ie Failed at call-stage due to a malformed fastq file).
+
+  instrument_name:
```

Comment:
@turbomam  not sure if you want to use the new deprecation way we talked about or if this should just be deleted.

Reply:
cc @mslarae13

Reply:
My vote is just deleted since it's the fork & we just deleted everything else.

Reply:
if you mean remove it from nmdc-schema production (main), then deprecation protocol we decided on. I think it would be a good spot for Sierra to help (if so)

Reply:
oops I already deleted it! but we can reconstruct a deprecation if you want @aclum

Reply:
Deleting is fine with me.